modules/mongodb/testdata/v5.0.0/serverStatus.go

```diff
@@ -0,0 +1,2255 @@
+package v5_0_0
+
+var V500ServerStatus = `
```

Comment:
Wow, that is a lot of data. Is it possible to query specific stats? Or it is either all or nothing?

Reply:
We can only disable `repl`, `metrics `, `locks` in command or maybe use aggregation to filter out unwanted data but this will transfer the load from the collector to the database itself. I am not 100% what is the correct balance, some extra network IO or some CPU power from the database itself.

Reply:
It is 2255 lines, how many if we disable those three? Or how many each of them adds?

Reply:
locks is about 38 lines of pretified json/bson and metrics is about 1189 lines of pretty json/bson or **685 values** and repl about 34 lines. Keep in mind that we do not collect them all. Just a portion.

Reply:
To add some more details, this big response is only for self deployed databases. Atlas reports quite a few less (150 lines of prettified json/bson) data and we product only 8 charts in this case. The other charts are only enabled if we get data back from the server at the appropriate sections.